src/components/course/course-header/CourseHeader.jsx

```diff
@@ -49,6 +50,25 @@ const CourseHeader = () => {
     config.EXPERIMENT_5_VARIANT_1_ID,
   );
   const hasSufficientReviewCount = courseReviews?.reviewsCount >= 5;
+  useEffect(() => {
+    if (hasSufficientReviewCount && isExperimentVariationA) {
+      sendEnterpriseTrackEvent(
+        enterpriseConfig.uuid,
+        'edx.ui.enterprise.learner_portal.course.viewedWithReviewsVariation',
+        {
+          course_key: course.key,
+        },
+      );
+    } else {
+      sendEnterpriseTrackEvent(
+        enterpriseConfig.uuid,
+        'edx.ui.enterprise.learner_portal.course.viewedWithoutReviewsVariation',
+        {
+          course_key: course.key,
+        },
+      );
+    }
+  }, [hasSufficientReviewCount, isExperimentVariationA, enterpriseConfig, course]);
```

Comment:
nit: would recommend `course.key` in the dependency array; that way, the effect only runs if/when the course key specifically changes.